lunes_cms/api/v1/serializers/feedback_serializer.py

```diff
@@ -23,6 +23,7 @@ class FeedbackSerializer(serializers.ModelSerializer):
         },
     )
 
+    # pylint: disable=raise-missing-from
```

Comment:
```suggestion
```
I think it makes sense to do as pylint suggest and use `except Exception as e` in combinaton with `raise ... from e`.